halo2_backend/src/plonk/verifier.rs

```diff
@@ -33,29 +33,28 @@ pub use batch::BatchVerifier;
 pub fn verify_proof_single<'params, Scheme, V, E, T, Strategy>(
```

Comment:
Since we're breaking the API with this PR, I propose the following change:
- `verify_proof_single` -> `verify_proof`
- `verify_proof` -> `verify_proof_multi`
And similarly for proving:
- `ProverSingle` -> `Prover`
- `Prover` -> `ProverMulti`

This is because the use case of calculating/verifying a single proof is more common than the batch proving/verifying.

Reply:
Done in https://github.com/privacy-scaling-explorations/halo2/pull/359/commits/bce0bebb934253392dffdf58c0caf64c17b04c8f and https://github.com/privacy-scaling-explorations/halo2/pull/359/commits/9fc59c100ed3e65f58c174853f8bd4d6c2bc8231